README.md

```diff
@@ -112,6 +112,136 @@ The script has been tested with the KALDI Git hash "7637de77e0a77bf280bef9bf484e
+        "example_id": "...",
+        "num_speakers": 2,
+        "speaker_id": ["...", "..."],
+        "gender": ["...", "..."],  # "male" or "female"
+        "kaldi_transcription": ["...", "..."],
+        "log_weights": [..., ...],  # weights of utterances before the are added
+        "num_samples": {
+          "original_source": [..., ...],
+          "observation": ...,
+        },
+        "offset": [..., ...]  # Offset of utterance start in samples
+        "snr": ...,
+        "audio_path": {
+          "original_source": ["...", "..."],
+          "speech_source": ["...", "..."],
+          "rir": ["...", "..."],
+          "speech_reverberation_early": ["...", "..."],
+          "speech_reverberation_tail": ["...", "..."],
+          "noise_image": "...",
+          "observation": "...",
+        }
+      }
+    }
+  }
+}
+```
+This file can be used to get all details for each example.
+To read it with python, we have some helper functions:
+<!-- To access one (or multiple) examples, you can use the following setup code: -->
+
+```python
+from sms_wsj.database import SmsWsj, AudioReader
+db = SmsWsj(json_path='.../sms_wsj.json')
+ds = db.get_dataset('train_si284')  # "train_si284", "cv_dev93" or "test_eval92"
+ds = ds.map(AudioReader((
+    'observation',
+    'speech_source',
+    # 'original_source',
+    # 'speech_reverberation_early',
+    # 'speech_reverberation_tail',
+    'speech_image',
+    # 'noise_image',
+    # 'rir',
+)))
+ds = ds.shuffle(reshuffle=True)
+# ds = ds.batch(batch_size)  # Create a list from `batch_size` consecutive examples
+# ds = ds.batch(my_collate_fn)  # e.g. sort the batch, pad/cut examples, move outer list to batch axis, ...
+# ds = ds.prefetch(4, 8)  # Use a ThreadPool with 4 threads to prefetch examples
+```
+
+Now you can access the examples with the dataset instance.
+You can iterate over the dataset (e.g. `for example in ds: ...`) or access examples by their ID, e.g. `ds['0_4axc0218_01kc020f']`
+(Access with an index (e.g. `ds[42]`) only works, when the dataset is not shuffled.).
+The audio files, that are requested from the `AudioReader` will be loaded on demand and will be available under the key `audio_data` in the `example`.
+
+If you want to reduce the IO, you can use the `scenario_map_fn`:
+```python
+from sms_wsj.database import SmsWsj, AudioReader, scenario_map_fn
+db = SmsWsj(json_path='.../sms_wsj.json')
+ds = db.get_dataset('cv_dev93')  # "train_si284", "cv_dev93" or "test_eval92"
+ds = ds.map(AudioReader((
+    'original_source',
+    'rir',
+)))
+ds = ds.map(scenario_map_fn)  # Calculates all signals from `original_source` and `RIR`
+ds = ds.shuffle(reshuffle=True)
+# ds = ds.batch(batch_size)  # Create a list from `batch_size` consecutive examples
+# ds = ds.batch(my_collate_fn)  # e.g. sort the batch, pad/cut examples, move outer list to batch axis, ...
+# ds = ds.prefetch(4, 8)  # Use a ThreadPool with 4 threads to prefetch examples
+```
```

Comment:
I would delete the commented out calls to lazy_dataset since they are unrelated to the database

Reply:
My idea was to provide enough code, that anyone can copy the code here and start, without looking into `lazy_dataset`.